netbox_branching/collapse_merge.py

```diff
@@ -0,0 +1,675 @@
+"""
```

Comment:
It seems like this file is left-over from the refactor, yeah?

Reply:
oops, yeah